.github/workflows/test-spec.yml

```diff
@@ -26,7 +26,10 @@ jobs:
           - {version: 1.3.17, admin_password: admin}
           - {version: 2.0.0, admin_password: admin}
           - {version: 2.15.0}
-          - {version: 2.16.0, hub: opensearchstaging, ref: '@sha256:50fbfe3b95c41e92a113ada3e80513ba4524dfc8a25dc6aaeff2bbe1e1145d5f'}
+          - version: 2.16.0
+            hub: opensearchstaging
+            ref: '@sha256:50fbfe3b95c41e92a113ada3e80513ba4524dfc8a25dc6aaeff2bbe1e1145d5f'
+            features: '-Dopensearch.experimental.feature.tiered_remote_index.enabled=true'
```

Comment:
This feels like just a parameter, either make this `opts` (for `OPENSEARCH_JAVA_OPTS`) or (this might be harder but nicer)

```yaml
experimental_features:
  - tiered_remote_index: true
```

Reply:
updated to opts